test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -40,6 +40,12 @@ public void execute() throws IllegalValueException {
 
         //Keyword matching a word in address: not matched
         assertFindCommandBehavior(new String[]{"Clementi"}, Collections.emptyList());
+
+        //Keyword matching a word in Tag: matched
+        assertFindCommandBehavior(new String[]{"Test"}, Arrays.asList(td.dan));
+
+        //Keywords matching both tag and name: matched
+        assertFindCommandBehavior(new String[]{"Test", "Amy"}, Arrays.asList(td.amy, td.dan));
```

Comment:
You could add another test for repeated keywords of the same tag (similar to the "Amy", Amy" test for name keywords)

Reply:
Will do. Thanks!